### PR TITLE
gin-cors update and Using go module instead of dep

### PR DESCRIPTION
**Pull request by scebar**
gin-cors update to v.1.3.0 
Using go module instead of dep

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -0,0 +1,8 @@
+module github.com/krakend-cors
+
+require (
+	github.com/devopsfaith/krakend v0.0.0-20190616151918-b11cce82be2d
+	github.com/devopsfaith/krakend-cors v0.0.0-20180906120953-c9b7eb26914f
```
Comment (kpacha) [inline go.mod]: the package should not depend on itself

File: go.mod [hunk 2]
```diff
@@ -0,0 +1,8 @@
+module github.com/krakend-cors
```
Comment (kpacha) [inline go.mod]: the package name should be `github.com/devopsfaith/krakend-cors`

**Comment by scebar** [comment]
hi @kpacha 
It's my mistake and i changed my pull request as you said. thx

File: go.sum [hunk 3]
```diff
@@ -0,0 +1,30 @@
+github.com/davecgh/go-spew v1.1.0/go.mod h1:J7Y8YcW2NihsgmVo/mv3lAwl/skON4iLHjSsI+c5H38=
+github.com/davecgh/go-spew v1.1.1/go.mod h1:J7Y8YcW2NihsgmVo/mv3lAwl/skON4iLHjSsI+c5H38=
```
Comment (kpacha) [inline go.sum]: why is this package required? it was not present on the `.lock` file

Comment (scebar) [reply go.sum]: I don't know. go.sum and go.mod are created by go module. Maybe a repo used depends on them

Comment (kpacha) [reply go.sum]: yes, but I can't find where it comes from

this the output for executing `goda graph -cluster -short ./... | dot -Tpng -o graph.png` on your branch
graph

this is not a major issue, but I'd keep the deps as clean as possible

Comment (scebar) [reply go.sum]: you are absolutely right. I checked it now using gin-contrib/cors repo.

**Comment by scebar** [comment]
hi @kpacha
Why dont u accept my pull request.

**Comment by scebar** [comment]
okey, thanks.

**Pull request merged**